apps/storybook/docs/d_planning/2_aanpak_v2_v3.stories.mdx

```diff
@@ -0,0 +1,170 @@
+import {Meta} from '@storybook/blocks';
+
+<Meta title="Planning/Aanpak v2 [2024] & v3 [2025]"/>
+
+# Aanpak v2 & v3
+
+## Inhoudstafel
+
+- [Waarom v2](#waarom-v2)
+- [Waarom Next Componenten](#waarom-next-componenten)
+- [Next Conventies](#next-conventies)
+- [v2 vs. v3](#v2-vs-v3)
+- [v2 Release](#v2-release)
+
+
+## Waarom v2
+
+De basis van de code die er is [dateert](/docs/historiek-toekomst--documentatie) van 2020. Er is toen vertrokken
+vanuit het idee de code van [Webuniversum v3](https://overheid.vlaanderen.be/webuniversum/v3/) as-is af te nemen.
+Dit had een aantal gevolgen:
+
+### Element complicatie
+
+Componenten zoals de DV [vl-ui-button](https://overheid.vlaanderen.be/webuniversum/v3/documentation/atoms/vl-ui-button)
+moeten als een uitbreiding op een native button geïmplementeerd worden. De UIG web component moet dus extenden van
+HTMLButtonElement. De rede is dat de scss van DV enkel werkt op een `button` tag, een custom component maken met als
+tag `vl-button` werkt niet (de css die DV voorziet is daarvoor niet correct). Er was wel de mogelijkheid geweest om de
+component te schrijven als wrapper: een `vl-button` die dan daarin een `button` rendert, maar die keuze is toen niet
+gemaakt.
+
+Die keuze, om dus de vl-button als een uitbreiding op een button te implementeren, heeft een aantal gevolgen:
+
+- afnemers moeten dit type componenten op een specifieke manier gebruiken: m.b.v. het `is=` attribuut, zie
+  [https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/is](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/is)
+- Safari gaat dit nooit ondersteunen, we moeten de polyfill `@ungap/custom-elements` voorzien om dat in Safari te
+  ondersteunen
+- bestaande web component frameworks: bvb. [Lit](https://lit.dev/) en [Stencil](https://stenciljs.com/docs/introduction)
+  ondersteunen het niet; overal LitElement (om consistent te zijn) als basis klasse gebruiken kan dus niet
+- omdat op die manier - aan native elementen - extra attributen toegevoegd worden door de implementatie, en om
+  tegelijkertijd aan de HTML5 standaard te voldoen, is er gekozen de nieuwe attributen te prefixen met 'data-'; dit is
+  dan (om consistentie redenen) bij alle componenten zo doorgevoerd - terwijl dat voor componenten met een custom-tag
+  geen verplichting (en eigenlijk niet de bedoeling) is
+
+Opmerking: om duidelijk te maken of een bepaalde component gebruikt moet worden via een `custom tag` of via het `is=`
+attribuut zijn in de nieuwe `@domg-wc` componenten (v1.x.x) alle componenten, die als uitbreiding op een native tag
+geïmplementeerd zijn, gegroepeerd in het `@domg-wc/elements` artifact.
```

Comment:
Eventueel link naar artifacts pagina > `[artifact](/docs/afnemen-artifacts--documentatie#domg-wcelements)]`

Reply:
Ik denk dat ik het daar niet deed omdat je die `@domg-wc/elements` naar 1 artifact kan linken maar `@domg-wc` componenten slaagt op al de rest (components / form / section / grid) en daar kan je dan weer niet 1 link bij zetten.

Er is op heel veel plaatsen de mogelijkheid om links toe te voegen, dat is niet perse een vergetelheid hoor. Ik probeer te kijken naar de context van de pagina om van iets een link te maken, enkel als het in die specifieke context relevant is maak ik er een link van. Maar zelfs dat is subjectief van Safari, polyfill of ungap maak ik ook geen link terwijl van Lit en Stencil wel.